README.md

```diff
@@ -21,7 +21,7 @@ This library requires you use Ruby 2.3 or above.
 
 ## Configuration
 
-```
+```rb
```

Comment:
Here the whole time I thought it had to be 'ruby'; 15 years later and you're still teaching me things.